server/controllers/projectController.js

```diff
@@ -1,5 +1,14 @@
-const Project = require("../models/projects");
-const User = require("../models/User");
+const Project = require('../models/projects');
+const User = require('../models/User');
+
+async function getProject(req, res) {
+  const project = await Project.findOne({ _id: req.params.projectId });
+  if (!project) {
+    return res.status(400);
+  }
+  const user = await User.findOne({ _id: project.owner }).select({ name: 1 });
+  res.send({ project, user });
```

Comment:
I think you can use the mongoDB populate to get the owner information inside of the project data

Reply:
Okay, modified to populate and changed on the front end.